specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintDefinition.json

```diff
@@ -416,6 +416,15 @@
           },
           {
             "$ref": "#/parameters/PublishedBlueprintVersionIdParameter"
+          },
+          {
+            "name": "publishedBlueprint",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/PublishedBlueprint"
+            },
+            "description": "Published Blueprint to create or update."
           }
```

Comment:
Note for other reviewers:
The blueprint backend has always supported this and it was always optional. So SDKs generated from swaggers with and without this change will work just fine.